src/git-rename-branch

```diff
@@ -85,6 +83,8 @@ function steps {
   echo delete_parent_entry "$target_branch_name"
   echo store_parent_branch "$destination_branch_name" "$parent_branch_name"
   echo "delete_all_ancestor_entries"
+
+  echo "delete_local_branch '$target_branch_name' force"
```

Comment:
Undoing this runs operations that require this branch to exist locally. Hence, it has to be created first thing in the undo operation. Hence it has to be deleted last.